chore(curriculum): remove wiki reference from cip (#46457)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/sedols.md

# --description--

> <abbr title="Stock Exchange Daily Official List">SEDOL</abbr> is a list requested internationally for its ability to serve as identifying number. It's uniquely assigning different codes for millions of data worldwide.

Reviewer: ```suggestion
<abbr title="Stock Exchange Daily Official List">SEDOL</abbr> is a list of securities identification numbers issued by the London Stock Exchange.
```
